src/pages/FormPageHoc.tsx

```diff
@@ -24,13 +24,21 @@ export type ValidateFormParams<P> = {
   formProps: P;
 };
 
-export const FormPageHoC = <P extends Record<string, unknown>>(
+interface FormPageHoCProps<P extends Record<string, unknown>> {
   FormComponent: (
     props: P & { errors: ValidationErrorItem[] | undefined }
-  ) => JSX.Element,
-  formProps: P,
-  validatePage?: (params: ValidateFormParams<P>) => ValidationErrorItem[]
-) => {
+  ) => JSX.Element;
+  formProps: P;
+  validatePage?: (params: ValidateFormParams<P>) => ValidationErrorItem[];
+  twoThirdsWidth?: boolean;
+}
+
+export const FormPageHoC = <P extends Record<string, unknown>>({
+  FormComponent,
+  formProps,
```

Comment:
can we make formProps optional so we dont have to pass {} in main.tsx?

Reply:
Not without the types complaining, I spent a while looking into it but didn't manage to make any headway